File: wrappers/mvc/demos/Kendo.Mvc.Examples/Controllers/Scatter_Charts/Pan_And_ZoomController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using Kendo.Mvc.Examples.Models;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class Scatter_ChartsController : Controller
    {
        public ActionResult Pan_And_Zoom()
        {
            return View(ChartDataRepository.SineInterval(-50, 50));
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/scattercharts/PanAndZoomController.java
package com.kendoui.spring.controllers.scattercharts;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import com.kendoui.spring.models.ChartDataRepository;

@Controller("dataviz-scatter_chart-pan_and_zoom-controller")
@RequestMapping(value="/scatter-charts/")
public class PanAndZoomController {
    @RequestMapping(value = "/pan-and-zoom", method = RequestMethod.GET)
    public String index(Model model) {
        model.addAttribute("data", ChartDataRepository.SineInterval());
     
        return "/scatter-charts/pan-and-zoom";
    }
}
